test/k8sT/manifests/fqdn-httpd.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: fqdn-httpd
+spec:
+  selector:
+    matchLabels:
+      app: fqdn-httpd
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: fqdn-httpd
+    spec:
+      hostNetwork: true
+      containers:
+      - name: httpd
+        image: docker.io/cilium/demo-httpd:latest
+        ports:
+        - containerPort: 80
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+          - labelSelector:
+              matchExpressions:
+              - key: app
+                operator: In
+                values:
+                - fqdn-httpd
+            topologyKey: "kubernetes.io/hostname"
```

Comment:
Isn't this basically a deamonset? :-)

Reply:
yes and no. In the case where there are more nodes I think the idea was to avoid the pods showing up all over the place. Outside that, it is one :) (unless deamonset pods don't allow ` --field-selector spec.nodeName=%s"` but I expect that would work fine).